va/va_fei_hevc.h

```diff
@@ -165,7 +165,9 @@ typedef struct _VAEncMiscParameterFEIFrameControlHEVC
     uint32_t      search_window             : 8;
     /** \brief number of internal MV predictors for IME searches */
     uint32_t      max_num_ime_search_center : 3;
-    uint32_t      reserved2                 : 5;
+    /** \brief fast intra prediction enabling bit */
+    uint32_t      fast_intra_mode           : 1;
```

Comment:
How is an application or middleware aware of this capability of the underlying driver? What's the result if application set this flag but the driver doesn't support fast intra mode?

Reply:
There will be the corresponding interface in MSDK API, and also have detail descriptions in MSDK doc. Driver will ignore the flag and the result will be no change if setting the flag in the unsupported driver.
I will update the code comments for more detailed descriptions.